packages/apps/graph-client/package.json

```diff
@@ -47,10 +46,10 @@
     "@rushstack/heft": "~0.50.6",
     "@types/heft-jest": "~1.0.3",
     "@types/lodash.isequal": "~4.5.6",
-    "@types/node": "^16.0.0",
+    "@types/node": "^20.4.2",
```

Comment:
What are the implications of this? We still support Node.js v16, so we might get in trouble with this?

Reply:
Yeah you're right, doesn't make sense to update this when we still support 16. I'll revert this.